packages/api/cms-api/src/generator/generate-crud.ts

```diff
@@ -891,11 +920,18 @@ function generateResolver({ generatorOptions, metadata }: { generatorOptions: Cr
         }
     }
 
+    imports.push({ name: "extractGraphqlFields", importPath: "@comet/cms-api" });
+    imports.push({ name: "SortDirection", importPath: "@comet/cms-api" });
+    imports.push({ name: "RequiredPermission", importPath: "@comet/cms-api" });
+    imports.push({ name: "AffectedEntity", importPath: "@comet/cms-api" });
+    imports.push({ name: "validateNotModified", importPath: "@comet/cms-api" });
+    imports.push({ name: "RootBlockDataScalar", importPath: "@comet/cms-api" });
+    imports.push({ name: "BlocksTransformerService", importPath: "@comet/cms-api" });
```

Comment:
Why this change? And why not do it for all imports? 😁

Reply:
For some reason eslint autofix failed to merge duplicated imports (from `@comet/cms-api`) for those, for the others it worked correctly.

> And why not do it for all imports?

yes, maybe we should switch to that technique